.github/workflows/enzyme-julia.yml

```diff
@@ -27,10 +27,8 @@ jobs:
           - x64
     timeout-minutes: 60 
     steps:
-    - uses: actions/checkout@v2
-      with:
-        fetch-depth: 1
-    - uses: actions/checkout@v2
+    - uses: actions/checkout@v3
+    - uses: actions/checkout@v3
```

Comment:
Do we even need the Enzyme.jl tests for this purpose here?? I think just removing them in their entirety is a much cleaner approach.